lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationSerializer.cs

```diff
@@ -145,18 +145,18 @@ public IConfiguration FromBase64String(string serializedConfig)
         public string ToString(IConfiguration c)
         {
             byte[] bytes = ToByteArray(c);
-            AvroConfiguration avroConf = AvroDeseriaize(bytes);
+            AvroConfiguration avroConf = AvroDeserialize(bytes);
             string s = JsonConvert.SerializeObject(avroConf, Formatting.Indented);
             return s;
         }
 
-        public IConfiguration FromString(string josonString)
+        public IConfiguration FromString(string jsonString)
         {
-            AvroConfiguration avroConf = JsonConvert.DeserializeObject<AvroConfiguration>(josonString);
+            AvroConfiguration avroConf = JsonConvert.DeserializeObject<AvroConfiguration>(jsonString);
             return FromAvro(avroConf);
         }
 
-        public AvroConfiguration AvroDeseriaizeFromFile(string fileName)
```

Comment:
Public method in non-example non-test code => needs to go through "Obsolete in 0.14" path. Next two methods are private, so they're fine